s3tests.conf.SAMPLE

```diff
@@ -111,7 +111,7 @@ secret_key = nopqrstuvwxyzabcdefghijklmnabcdefghijklm
 display_name = testx$tenanteduser
 
 # tenant user_id set in vstart.sh
-user_id = 9876543210abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+user_id = testx$9876543210abcdef0123456789abcdef0123456789abcdef0123456789abcdef
```

Comment:
does this not break other tests?

if you need this tenant-user's tenant name for a test case, that's exposed by `get_tenant_name()`

Reply:
ok, thanks. so i guess this change makes vstart testing consistent with teuthology :+1: